LanguageFeatures/Patterns/object_A07_t06.dart

```diff
@@ -20,7 +20,7 @@
 /// pattern with the getter name omitted (see name inference below).
 ///
 /// @description Check that wildcard `_` can be used in Object pattern in a
-/// declaration context. Getter is called, but its value is ignored
+/// declaration context. Getter is not called
```

Comment:
I don't understand this. With an objectPattern it would hardly make sense to omit the getter invocation: `Square(areaAsInt: _)` simply doesn't do anything extra compared to `Square()`, and `Square(areaAsNullable: _ as int)` wouldn't have anything to test in the castPattern if we weren't calling the getter (based on the reasoning that it would be passed into a wildcard pattern anyway).

[Here](https://github.com/dart-lang/language/blob/master/accepted/future-releases/0546-patterns/feature-specification.md#:~:text=in%20source%20order%3A-,Call%20the%20getter,-with%20the%20same)'s a location in the patterns feature spec that seems consistent with my expectation that the getter _should_ be invoked.

I think it's harder to tell for a mapPattern. I think there was a discussion about not calling operator `[]` in the case where a patternField had the form `someKey: _`, but I can't spot anything in the fetaure spec to confirm this. But that's a separate question, and I do think the getter should be called when the patternField occurs in an objectPattern.

Reply:
Created an SDK issue https://github.com/dart-lang/sdk/issues/51029